LocalPackages/NetworkProtectionMac/Sources/NetworkProtectionUI/Extensions/LottieView+withIntro.swift

```diff
@@ -28,14 +28,14 @@ extension LottieView where Placeholder: View {
         let loopEndFrame: AnimationFrameTime
     }
 
-    public func playing(withIntro timing: LoopWithIntroTiming, isAnimating: Binding<Bool> = .constant(true)) -> Lottie.LottieView<Placeholder> {
+    public func playing(withIntro timing: LoopWithIntroTiming, isAnimating: Bool = true) -> Lottie.LottieView<Placeholder> {
```

Comment:
This method wasn't using the binding at all, just the wrapped value... so we might as well just request a `Bool` instead and offer a cleaner method.